.github/workflows/minimal.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           ssh -i .test-key -v -p 2222 -l test-user 127.0.0.1 -o StrictHostKeyChecking=accept-new -o UserKnownHostsFile=/tmp/openssh-rs/known_hosts whoami
         name: Test ssh connectivity
       - run: |
+          sleep 30
```

Comment:
I added this because the ci randomly failed.

The `openssh-server` container actually calls `apk` to install some softwares required to fetch the mod `DOCKER_MODS: linuxserver/mods:openssh-server-ssh-tunnel` and then downloads and applies the mod on the fly before starting `sshd` process, so the startup time varies and sometimes it just falls.

I tried `sleep 5` but it still failed so I changed that to `sleep 30`.

I guess I can try `sleep 15` as 15s should be enough.

Reply:
Done!

Reply:
To maybe get rid of a hard coded sleep one could use `ssh-keyscan` to probe the ssh server. I use `until ssh-keyscan "$IP" > /root/.ssh/known_hosts; do sleep 1; done` in one of my own CIs to wait for the remote host to accept ssh connections.

Reply:
Nice!
I will have a try.

Reply:
@Huanzo Thank you very much!
I've now fixed it by using a proper solution, combining your solution with a `timeout`.